test/functional/p2p_invalid_messages.py

```diff
@@ -3,12 +3,14 @@
 # Distributed under the MIT software license, see the accompanying
 # file COPYING or http://www.opensource.org/licenses/mit-license.php.
 """Test node responses to invalid network messages."""
+
```

Comment:
nit: extra newline here?
```suggestion
```

Reply:
Using (https://www.python.org/dev/peps/pep-0257/#id17)[PEP 257] as a basis, this could be argued either way.  I like this better, and it's consistent with the 7 or so other test files that I just checked.